relay/polkadot/src/lib.rs

```diff
@@ -1886,6 +1889,18 @@ pub mod migrations {
 		}
 	}
 
+	/// Enable the elastic scaling node side feature.
+	///
+	/// This is required for Coretime to ensure the relay chain processes parachains that are
+	/// assigned to multiple cores.
+	pub struct EnableElasticScalingNodeFeature;
+	impl OnRuntimeUpgrade for EnableElasticScalingNodeFeature {
+		fn on_runtime_upgrade() -> Weight {
+			let _ = Configuration::set_node_feature(RawOrigin::Root.into(), 1, true);
```

Comment:
Why are we not using the [constant](https://github.com/paritytech/polkadot-sdk/blob/523e62560eb5d9a36ea75851f2fb15b9d7993f01/polkadot/primitives/src/v7/mod.rs#L2036)?

Reply:
It's currently in staging, felt that we don't want to use any of the staging stuff here.

Reply:
Hmm - don't we have v7 already in the current release? In any case, seems like we would be using staging stuff anyways, just hiding it by using a magic number? Also I think this can wait for the next polkadot-sdk release, where this should be v7 then.

Reply:
That makes sense, but this needs to go in same release with core time.